presto-testing/src/main/java/io/prestosql/testing/AbstractTestDistributedQueries.java

```diff
@@ -407,6 +424,7 @@ public void testAddColumn()
         assertUpdate("CREATE TABLE " + tableName + " AS SELECT CAST('first' AS varchar) x", 1);
 
         assertQueryFails("ALTER TABLE " + tableName + " ADD COLUMN x bigint", ".* Column 'x' already exists");
+        assertUpdate("ALTER TABLE " + tableName + " ADD COLUMN IF NOT EXISTS x bigint");
```

Comment:
call this `assertUpdate(...)` twice

Reply:
updated. For knowledge purpose, can you tell me why should we call "assertUpdate("ALTER TABLE " + tableName + " ADD COLUMN IF NOT EXISTS x bigint");" twice?

Reply:
Oh my bad. My point was to run it twice so once it creates a column and then it skips column creation (suppress the error0. But here column `x` already exists. So to make it work here we need to use different column name than `x`.

Also it would be better to use something else than `x` something that would express the test case. Maybe `x` -> `testing_add_column_if_not_exists`?

Reply:
updated! Already add a statement to test can add column even with "if exists" statement, so made a duplicate of which.